src/geometry.cpp

```diff
@@ -79,4 +81,25 @@ namespace geometry {
 	  return v;
 
 	}
+
+	std::vector<Cell> raytraceLine_conversion(const Cell& start, const Cell& end) {
+
+	  // Get the interpolated pixels bresenham
+	  const auto pixelsAsInts = bresenham_conversion(start.x, start.y, end.x, end.y);
```

Comment:
variable names should be snake case

Reply:
Make an overload to bresenham to take `Cell`, unless you would rather do that in a follow up pr.
Now that you have `Cell`, I'm not sure why bresenham should ever take raw x y values.

Reply:
Done